src/marqo/tensor_search/add_docs.py

```diff
@@ -192,4 +196,51 @@ def determine_document_dict_field_type(field_name: str, field_content, mappings:
         else:
             raise base_exceptions.InternalError(f"Invalid dict field type: '{mappings[field_name]['type']}' for field: '{field_name}' in mappings. Must be one of {[t.value for t in enums.MappingsObjectType]}")
     else:
-        return None
\ No newline at end of file
+        return None
+    
+
+
+def determine_text_prefix(request_level_prefix: str, index_settings: IndexSettings, prefix_type: str) -> str:
```

Comment:
determine_text_chunk_prefix could be a better name, to specify that it's for chunks.

Reply:
Ah I see, if this is both for query and chunk prefixes, it's best we move this to a different module. Something in the `core` dir would be good.